src/Microsoft.Tye.Hosting/DockerRunner.cs

```diff
@@ -468,9 +468,15 @@ void Log(string data)
                         service.Logs.OnNext(data);
                     }
 
+                    var arguments = $"build \"{docker.DockerFileContext?.FullName}\" -t {dockerImage} -f \"{docker.DockerFile}\"";
+                    foreach (var buildArg in docker.BuildArgs)
+                    {
+                        arguments += $" --build-arg {buildArg.Key}={buildArg.Value}";
```

Comment:
nit: StringBuilder is probably a good idea here as this will new up a string for each build arg.

Reply:
Done